staging/storage/minio/minio-distributed-statefulset.yaml

```diff
@@ -1,12 +1,17 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1beta2
 kind: StatefulSet
 metadata:
   name: minio
 spec:
+  selector:
+    matchLabels:
+      app: minio
   serviceName: minio
   replicas: 4
   template:
     metadata:
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
```

Comment:
I'm curious why you needed to add this. This annotation doesn't do anything I think.

Reply:
yes, this annotation has been deprecated, seen https://github.com/kubernetes/charts/pull/2333

Reply:
can you remove from here too?